central/complianceoperator/v2/profiles/datastore/datastore_impl_test.go

```diff
@@ -300,6 +309,107 @@ func (s *complianceProfileDataStoreTestSuite) TestGetProfileCount() {
 	s.Require().Equal(2, count)
 }
 
+func (s *complianceProfileDataStoreTestSuite) TestGetProfilesNames() {
+	// make sure we have nothing
+	profileIDs, err := s.storage.GetIDs(s.hasReadCtx)
+	s.Require().NoError(err)
+	s.Require().Empty(profileIDs)
+
+	rec1 := getTestProfile(profileUID1, "ocp4", "1.2", testconsts.Cluster1, 0)
+	rec2 := getTestProfile(profileUID2, "rhcos-moderate", "7.6", testconsts.Cluster1, 0)
+	rec3 := getTestProfile(profileUID3, "a-rhcos-moderate", "7.6", testconsts.Cluster1, 0)
+	rec4 := getTestProfile(uuid.NewV4().String(), "ocp4", "1.2", testconsts.Cluster2, 0)
+	rec6 := getTestProfile(uuid.NewV4().String(), "a-rhcos-moderate", "7.6", testconsts.Cluster2, 0)
+
+	s.Require().NoError(s.dataStore.UpsertProfile(s.hasWriteCtx, rec1))
+	s.Require().NoError(s.dataStore.UpsertProfile(s.hasWriteCtx, rec2))
+	s.Require().NoError(s.dataStore.UpsertProfile(s.hasWriteCtx, rec3))
+	s.Require().NoError(s.dataStore.UpsertProfile(s.hasWriteCtx, rec4))
+	s.Require().NoError(s.dataStore.UpsertProfile(s.hasWriteCtx, rec6))
+
+	clusterIDs := []string{testconsts.Cluster1, testconsts.Cluster2}
+
+	testCases := []struct {
+		desc           string
+		query          *apiV1.Query
+		clusterIDs     []string
+		testContext    context.Context
+		expectedRecord []string
+		expectedCount  int
+	}{
+		{
+			desc:           "Cluster 1 - Full access",
```

Comment:
IntelliJ supports running a single test case from the IDE, if you use a description as the key in a `map[string]struct{}` as shown here, for example: https://github.com/stackrox/stackrox/blob/c0df026db7b99a0c9abdcd6dc479b4e38de30ad4/image/embed_charts_test.go#L96-L110

This does not need to be changed in this PR though.

Reply:
oh that is good to know.